res/values/arrays.xml

```diff
@@ -27,7 +27,8 @@
     <item>neo2</item>
     <item>bone</item>
     <item>jcuken_ua</item>
-    <item>bangla</item>
+    <item>bengali_national</item>
+    <item>bengali_provat</item>
```

Comment:
Are these the unambiguous names for these layouts ?

Layouts are being renamed internally in https://github.com/Julow/Unexpected-Keyboard/pull/347 and good names were not yet found for the first one. Help needed.

Reply:
Yes, these Bengali names are unambiguous; these names are provided by layout designers.

If you are talking about layouts above Bengali, I can confirm both `neo2` and `jcuken_ua` are unambiguous but couln't find anything about `bone` layout. References: [neo2](https://en.wikipedia.org/wiki/File:Neo2_keyboard_layout.svg) and [jcuken_ua](https://en.wikipedia.org/wiki/JCUKEN).

Reply:
Thanks!

(I was asking for the two bengali layouts, [bone](https://web.archive.org/web/20180721192908/http://wiki.neo-layout.org/wiki/Bone) is an older variant of neo2)